espnet2/utils/plot_sinc_filters.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
+#  2020, Technische Universität München; Nicolas Lindae, Ludwig Kürzinger
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
```

Comment:
The path is not correct.
Move this script to pyscripts.

Reply:
moved `espnet2/utils/plot_sinc_filters.py`
to `egs2/TEMPLATE/asr1/pyscripts/utils/plot_sinc_filters.py`.
Is that the correct path?

Reply:
Yes, please.